pkg/volume/nfs/nfs.go

```diff
@@ -66,9 +68,29 @@ func (plugin *nfsPlugin) Name() string {
 	return nfsPluginName
 }
 
+func findMount() bool {
```

Comment:
is `isSupported` or `hasOSSupport` a better name?  What exactly are you testing?